doc/nrf/introduction.rst

```diff
@@ -9,9 +9,24 @@ Introduction
 
 
 The |NCS| is a scalable and unified software development kit for building low-power wireless applications based on the Nordic Semiconductor nRF52, nRF53, nRF70, and nRF91 Series wireless devices.
-It offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
+It supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.
 
-It integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications, samples, and protocol stacks such as Bluetooth® Low Energy, Bluetooth mesh, Matter, Thread/Zigbee, Wi-Fi®, and LTE-M/NB-IoT/GPS, TCP/IP.
-It also includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+The |NCS| has the following distinguishing features:
 
-The |NCS| supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.
+**Advanced middleware and security**
+   The |NCS| includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+
+**Based on Zephyr**
+   The |NCS| integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications and samples.
+
+**Extensible framework**
+   The |NCS| offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
+
+**Pre-certified libraries**
+   The |NCS| is optimized for pre-certified libraries, including SoftDevice, Matter, and Thread.
+
+**Robust connectivity support**
+   The |NCS| supports a wide range of connectivity technologies, such as Ant, Bluetooth® Low Energy, Bluetooth mesh, LTE-M/NB-IoT/GPS, Matter, Sidewalk, TCP/IP, Thread, and Wi-Fi®.
```

Comment:
"In addition to connectivity technologies provided by Zephyr [list] it supports [list]".